### ISSUE TITLE
JAWS does not support alt text for CSS pseudo content content with url image

### ISSUE THREAD

**Issue by mraccess77** [FreedomScientific/VFO-standards-support#367]
###Summary
JAWS does not support alt text for CSS pseudo content content with url image on calculated accessible name.  JAWS will speak CSS content when url is not used -- but not when URL is used.

Example:
button::before {
  content: url(./img/star.png) / "New!";
}

### Expected result
The text "New!" is announced.

### Actual result
The text "New!" is not announced.

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

#### JAWS version and build number
2020 - February Update

 
#### Browser and version:
Chrome

**Comment by JAWS-test** [comment]
This possibility of alt text is so far only specified in the [CSS Working Draft](https://www.w3.org/TR/css-content-3/#alt). No idea whether this is already so final that AT should implement it

**Comment by JAWS-test** [comment]
I have tested the problem with Chrome 79.0.3945.130 and JAWS 2020, but the problem does not occur: the alternative text is output correctly

**Comment by cblouch** [comment]
Looks like it's working with buttons but not links on JAWS 2020.2001.70. For example:

<a href="secret.html" id=b1>link</a>

and then in the CSS

#b1:after {
content: url(test.jpg) / "this is alt text";
}

Does not seem to matter if I'm using :before or :after or whether the rule is based off the ID as shown above or applied to all anchors.

**Comment by JAWS-test** [comment]
True, for links the URL of the link is output if the pseudo content is a graphic, although Chrome transmits the Accessible Name correctly (see https://codepen.io/jaws-test/pen/eYNNdro). I had tested with buttons, because the example of @mraccess77 referred to buttons

**Comment by stevefaulkner** [comment]
noted that when virtual cursor turned off and the link is focused, it announces correctly. Also noted in chrome the correct accessible name is present for both button and link.
[test case](https://cdpn.io/stevef/debug/RwPPGOZ)

**Comment by JAWS-test** [comment]
Related: https://github.com/FreedomScientific/VFO-standards-support/issues/297

**Comment by JeremiahRogersVispero** [comment]
This works correctly in JAWS 2026.2606.132.
